### PR TITLE
fix reset num edge case

### PR DESCRIPTION
**Pull request by aleksandarbos**
## Code review checklist

- [x] Private platform tests at `core/tests/test_python-docx` are updated and passing
- [x] If this change is going to be deployed on Cloudsmith after merge, python-docx version number should be increased 

[commit messages]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: docx/oxml/numbering.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def get_preceding_paragraphs_numIds(p, p_ilvl, p_numId):
                     if prev_p_ilvl < p_ilvl and (prev_p_numId == p_numId or
```
Comment (balsa-sarenac) [inline docx/oxml/numbering.py:192]: While testing this I've also put the same equation in this if check. Everything worked. We should discuss it probably.

**Review by balsa-sarenac — APPROVED** [review]
